src/tools/ReadyToRun.SuperIlc/PerfEventSourceListener.cs

```diff
@@ -64,53 +65,65 @@ public PerfEventSourceListener(TraceEventSession traceEventSession, int totalWar
         // For all of the events below, we only want to process them after the warmup is complete. We can't just start this listener after
         // we have started the warmup because those runs might take some time, and we don't want to erroneously process warmup events when
         // trying to measure the real runs.
-        traceEventSession.Source.Dynamic.AddCallbackForProviderEvent(providerName, "Loading/Start", delegate (TraceEvent traceEvent)
+        traceEventSession.Source.Dynamic.AddCallbackForProviderEvent(providerName, "LoadingPhase/Start", delegate (TraceEvent traceEvent)
```

Comment:
Nit: We can just keep the names the way they are, without "Phase"

Reply:
I needed a way to distinguish between the wall clock JIT time and the total JIT time, so I added a new JitMethod event. I renamed the JitStart/JitStop to JitSectionStart/JitSectionStop to make the difference a bit clearer. Should I stick with the old names?